app/views/application_letters/_application_selective.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can? :edit_applicants, Event and not (application_letter.event.application_status_locked == true) %>
+<% if (can? :edit_applicants, Event) && (application_letter.event.phase == :selection) %>
```

Comment:
I think the "not (application_letter.event.application_status_locked == true)" should stay, since we want to lock it after sending emails